lib/foreman_discovery/engine.rb

```diff
@@ -39,7 +39,9 @@ class Engine < ::Rails::Engine
 
     # Add any db migrations
     initializer "foreman_discovery.load_app_instance_data" do |app|
-      app.config.paths['db/migrate'] += ForemanDiscovery::Engine.paths['db/migrate'].existent
```

Comment:
btw, why is this code duplicated in 151?
I just went with it because I didn't want to dive in, but it seems like this might be an error